Графика для всех страниц

Schtock opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2245591/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2245591-sedona-36/2/

Schtock committed:
выносит лого из блока нав

Schtock committed:
добавляет звезды к карточкам гостиниц

Schtock committed:
делает 3 точки в пагинации ссылкой

Schtock committed:
заменяет ul в пагинации на ol

Schtock committed:
заменяет ссылку "В избранное" на кнопку

Schtock committed:
Merge branch 'master' of https://github.com/Schtock/2245591-sedona-36

Schtock committed:
экспортирует графику в рабочий каталог

Schtock committed:
добавляет контентные изображения в разметку

Schtock committed:
добавляет логотип в шапке на страницу каталога

Dima-Dim commented on index.html:
```diff
@@ -51,7 +51,7 @@ <h2 class="visually-hidden">Преимущества</h2>
             <h3 class="advantages-title">Настоящий городок</h3>
             <p class="advantages-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
-            <img class="advantages-image" src="" alt="Вывешенные флаги в городке">
+            <img class="advantages-image" src="images/advantages/advantages-photo-1.jpg" width="800" height="384" alt="Вывешенные флаги в городке">
```
https://up.htmlacademy.ru/htmlcss/36/module/2/item/19

Dima-Dim reviewed (changes requested):
Убрать из разметки декоративные изображения.

Schtock committed:
убирает иконки из секции benefits

Schtock committed:
Заменяет лого академии и соц.сетей в футере на svg

Изменения нужны для возможности изменения состояний логотипов при наведении и клике.

Schtock committed:
Заменяет лого академии и соц.сетей в футере на svg

То же, что и на главной, но на странице каталога.
Изменения нужны для возможности изменения состояний логотипов при наведении и клике.

Dima-Dim commented on catalog.html:
```diff
@@ -242,23 +242,23 @@ <h2 class="visually-hidden">Подписка на рассылку</h2>
       </section>
     </main>
-    <footer class="main-footer">
+<footer class="main-footer">
```
Что случилось с отступом?

Dima-Dim commented on index.html:
```diff
@@ -82,17 +82,17 @@ <h2 class="visually-hidden">Достоинства</h2>
           <li class="benefits-item">
             <h3 class="benefits-title">Жильё</h3>
-            <img class="benefits-image" src="images/benefits-icons/benefits-accomodation.svg" width="75" height="72" alt="Логотип Жильё">
+            <div class="accomodation-icon"></div>
```
https://up.htmlacademy.ru/htmlcss/36/criteries#HTML-06

Schtock replied:
Дивы убрал, но не уверен насчет того, как правильно сделать.
Сейчас добавил заголовкам по второму классу, каждый из которых должен будет через CSS добавлять свою иконку.
Такой вариант подходит?

Dima-Dim replied:
Не знаешь потому, что вы это ещё не проходили. Сейчас для этих изображений в коде ничего быть не должно. Вам расскажут о том, как правильно это сделать в разделе про декоративные элементы.

Schtock committed:
Добавляет пропущенный отступ в футере

Schtock committed:
Убирает пустые дивы
